pkg/syncproto/sync_proto.go

```diff
@@ -280,11 +281,20 @@ func (s SerializedUpdate) ToUpdate() (api.Update, error) {
 				"Failed to parse value.")
 		}
 	}
+	revStr := ""
+	switch r := s.Revision.(type) {
+	case string:
+		revStr = r
+	case int:
```

Comment:
Would `fmt.Sprintf(s.Revision)` be good enough here?  I have uncertainty about whether etcd returns a 32 or 64-bit int and whether it's signed or not.

Reply:
Good idea!